### PR TITLE
Document and normalize worker CFC proto fields

### PR DESCRIPTION
**Pull request by werkt**
Provide an explanation of the functionality and possible benefits of
hex_bucket_levels.
Normalize controls and support for hex_bucket_levels and
file_directories_index_in_memory in operationqueue worker.

### CODE REVIEW

File: examples/worker.config.example [hunk 1]
```diff
@@ -46,6 +46,30 @@ root: "/tmp/worker"
...
+# the default, unspecified value is 0, to create all digests in the cas path itself
+# for each level > 0, a directory is created for all byte representations, 00-ff, with
+# as many sublevels as requested.
+#
+# This can provide a performance benefit for file access and manipulation for filesystems
+# which have problems with directories with many (e.g. millions) of files in them.
+# The CFC creates these directories upfront, resulting in a delay while each is verified or
+# created. Be cautious with large values, as the subdivision is 256^n
+#
+cas_cache_hex_bucket_levels: 0
```
Comment (walles) [inline examples/worker.config.example:71]: Shouldn't this be just `hex_bucket_levels`? Maybe I'm missing something obvious though, that has happened before...

File: examples/shard-worker.config.example [hunk 2]
```diff
@@ -69,6 +69,17 @@ cas: {
...
+    # create entry files in a directory for each leveled leading byte of the hash
+    # the default, unspecified value is 0, to create all digests in the cas path itself
+    # for each level > 0, a directory is created for all byte representations, 00-ff, with
+    # as many sublevels as requested.
+    #
+    # This can provide a performance benefit for file access and manipulation for filesystems
+    # which have problems with directories with many (e.g. millions) of files in them.
+    # The CFC creates these directories upfront, resulting in a delay while each is verified or
+    # created. Be cautious with large values, as the subdivision is 256^n
+    # hex_bucket_levels: 0
```
Comment (walles) [inline examples/shard-worker.config.example:82]: Shouldn't this match `worker.config.example`? Right now they use different variable names.

File: examples/worker.config.example (hunk 1 above)
Comment (werkt) [reply examples/worker.config.example:71]: The operationqueue worker does not use the cas configurations that are available in the shard worker - no support for delegation, and no containerization of the configs. There will likely be efforts to singularize this stream, and create one worker that can satisfy both behaviors with just singular config differences. This is a stopgap until that time, when a cas specification is sufficient and minimal.

File: examples/shard-worker.config.example (hunk 2 above)
Comment (werkt) [reply examples/shard-worker.config.example:82]: Situation as with above.

**Pull request merged**